test/utest.h

```diff
@@ -66,7 +66,12 @@ typedef uint64_t utest_uint64_t;
 #define UTEST_C_FUNC
 #endif
 
-#if defined(_MSC_VER)
+#if defined(_WIN32)
```

Comment:
I think this is fine for now - but as you mentioned on the PR it'd be great if you'd land a similar PR to utest.h to enable MINGW testing there?

Reply:
Yes I can try that. It depends on how long the task is going to be.